Makefile

```diff
@@ -158,6 +158,13 @@ cover: nocover
 	mv cover-nogen.out cover.out
 	go tool cover -html=cover.out -o cover.html
 
+.PHONY: cover-otel
+cover-otel:
+	@echo pre-compiling OTEL tests
+	@cd ${OTEL_COLLECTOR_DIR} && time go test -i $(shell cd ${OTEL_COLLECTOR_DIR} && go list ./...)
+	@cd ${OTEL_COLLECTOR_DIR} && ROOT_PKG=github.com/jaegertracing/jaeger/cmd/opentelemetry ../../scripts/cover.sh $(shell cd ${OTEL_COLLECTOR_DIR} && go list ./...)
```

Comment:
Do we use `.nocover` trick in cmd/opentelemetry anywhere?  If not then it's better to run normal `go test`, which is already capable of aggregating coverage across packages

Reply:
no we don't at the moment. I wanted to be aligned with the core module. I can switch to `go test`